packages/cactus-plugin-satp-hermes/src/main/typescript/generated/proto/cacti/satp/v02/common/health_pb.ts

```diff
@@ -1,4 +1,4 @@
-// @generated by protoc-gen-es v1.7.2 with parameter "target=ts"
+// @generated by protoc-gen-es v1.8.0 with parameter "target=ts"
```

Comment:
Let us please fix the latest working version of the generator in package.json (if not done yet)